source/includes/common-bucket-notifications.rst

```diff
@@ -0,0 +1,34 @@
+.. start-bucket-notification-find-arn
+
+.. admonition:: Identifying the ARN for your bucket notifications
+   :class: note
+
+   You defined the name to assign to the target ARN for your bucket notifications when creating the endpoint in step 1.
+   The command options below allow you to find the entire target ARN.
+   
+   **Review the JSON output**
+   
+   #. Copy and run the following command, replacing ``ALIAS`` with the :ref:`alias <alias>` of the deployment.
+
+      .. code-block:: shell
+         :class: copyable
+
+         mc admin info --json ALIAS
+
+   #. In the JSON output, look for the key ``info.sqsARN``.
+   
+      The ARN you need is the value of that key, such as |ARN|
```

Comment:
Same thought specifying it's the ARN that matches both the name + service as specified in step 1